strings/Java String Reverse/Solution.java

```diff
@@ -0,0 +1,21 @@
+import java.lang.String;
+import java.lang.StringBuilder;
+import java.util.Scanner;
+
+public class Solution {
+  
+  private static String checkIfPolindrom(String inputString) {
+    StringBuilder reversedString = new StringBuilder(inputString);
+    if (inputString.equals(reversedString.reverse().toString()) == true) {
```

Comment:
No need for `== true`. You'll never need it as it converts `true` to `true` and `false` to `false`, so basically does nothing :)